tests/python/relay/test_cpp_build_module.py

```diff
@@ -126,3 +155,4 @@ def test_fp16_conversion(target, dev):
     test_basic_build()
     test_fp16_build()
     test_fp16_conversion()
```

Comment:
Do we need to add `test_bf16_conversion` as fp16?

Reply:
I think it's already done in https://github.com/apache/tvm/blob/14d0187ce9cefc41e33aa30b55c08a75a6711732/tests/python/unittest/test_target_codegen_llvm.py#L718-L742